airbyte-integrations/bases/base-normalization/unit_tests/resources/edge_cases_catalog_expected_top_level.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
is this file being used?

Reply:
yes it is used and refer as the parameter "edge_cases_catalog"  in airbyte-integrations/bases/base-normalization/unit_tests/test_stream_processor.py